core/src/main/java/org/testng/xml/XmlInclude.java

```diff
@@ -32,10 +37,14 @@ public XmlInclude(String n, int index) {
 
   public XmlInclude(String n, List<Integer> list, int index) {
     m_name = n;
-    m_invocationNumbers = list;
+    m_invocationNumbers = new HashSet<>(list);
     m_index = index;
   }
 
+  public void setConstructorOrMethod(ConstructorOrMethod m_method) {
```

Comment:
@PavelSakharchuk - I think we can now revert this change since its not going to be used in the `FailedReporter` right ?

Reply:
I have done :)